lib/test/Test/Pos/Txp/Toil/UtxoSpec.hs

```diff
@@ -266,6 +270,13 @@ scriptTxSpec = describe "script transactions" $ do
                 alwaysSuccessValidator
                 (const witness)
 
+        it "validator script provided in witness is greater \
+           \than adopted script version" $ do
+            let witness = ScriptWitness intValidatorWithVersionOne intValidatorWithVersionOne
```

Comment:
This is actually a good test for one more reason: it checks that the version checks happens before running the scripts. It'd be nice to add an explicit comment about that here.